cypress/e2e/components/todos/conversion-filing.cy.ts

```diff
@@ -0,0 +1,42 @@
+context('TODOs -> Conversion todo task and filing task', () => {
+  it('Conversion todo item is rendered for staff account', () => {
+    cy.visitBusinessDashFor('businessInfo/ben/active.json', undefined, false, 'taskConversion.json', [], true)
+
+    cy.get('[data-cy="header_todo"]').should('exist')
+    cy.get('[data-cy="todoItemList"]').should('exist')
```

Comment:
can you update your cy selectors to follow the hierarchy we have in place?

So instead of selecting everything from root do it like:
```
cy.get('[data-cy="header_todo"]')
  .should('exist')
  .find('[data-cy="todoItemList"]')
  .should('exist).as('todoList')`
```
then below you could do:
```
cy.get('@todoList').find('[data-cy="todoItem-header-conversion"]').should('have.length', 1)
...
```

I this makes it a lot easier to understand how the page works / navigate failures in the tests and it allows your test to be more specific in many cases